plugins/pagerduty/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@backstage/plugin-pagerduty",
+  "version": "0.2.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/core": "^0.3.0",
+    "@backstage/theme": "^0.2.1",
+    "@backstage/catalog-model": "^0.2.0",
+    "@backstage/test-utils": "^0.1.2",
+    "@backstage/plugin-catalog": "^0.2.1",
```

Comment:
I think we should see if we could remove this dependency. We are importing a visual component from this package, and that should probably be moved out to `core` at some point. But we can do that later.